src/main/java/carpet/commands/LogCommand.java

```diff
@@ -201,15 +197,19 @@ private static int subscribePlayer(ServerCommandSource source, String player_nam
             Messenger.m(source, "r Unknown logger: ","rb "+logname);
             return 0;
         }
-        LoggerRegistry.subscribePlayer(player_name, logname, option);
-        if (option!=null)
-        {
-            Messenger.m(source, "gi Subscribed to " + logname + "(" + option + ")");
-        }
-        else
-        {
-            Messenger.m(source, "gi Subscribed to " + logname);
+        if (!Arrays.asList(LoggerRegistry.getLogger(logname).getOptions()).contains(option)) {
```

Comment:
This would remove the possiblility for loggers to have more flexible custom options, since the `option` field might only be a suggestion argument set for players

e.g. In carpet tis addition, ticket logger or item/xp orb loggers allow player to use `,` separated arguments as the logger argument, so it can display what exactly player wants to know

Reply:
Ah gnembon has also mentioned this issue above xd

Reply:
Yeah that's what I realized. My thought was that I could make it so that the player could pass some specific value into the loggers option (for example an empty String Array was what I thought of but you could probably do this better) which would signify that it can have any input. I think I'm gonna try to rework it so that it doesn't break any of the current loggers but also allows for them to optionally be restrictive.